test/integration/router_test.go

```diff
@@ -0,0 +1,301 @@
+// +build integration,!no-docker,docker
```

Comment:
Oh man, I can't believe that this is what it was.  Successful tests locally.  You receive all my internet points for the day.